queue/test/fiskaltrust.Middleware.Storage.AcceptanceTest/AbstractCopyPayloadRepositoryTests.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Threading.Tasks;
+using fiskaltrust.Middleware.Contracts.Repositories.FR;
+using fiskaltrust.Middleware.Contracts.Repositories.FR.TempSpace;
+using Xunit;
+
+namespace fiskaltrust.Middleware.Storage.AcceptanceTest
+{
+    public abstract class AbstractCopyPayloadRepositoryTests
+    {
+        protected abstract Task<IJournalFRCopyPayloadRepository> CreateRepository();
+        protected abstract Task DisposeDatabase();
+
```

Comment:
We need to add a test that verifies if `GetCountOfCopiesAsync` returns the correct number.

Reply:
I just [created it](https://github.com/fiskaltrust/middleware/pull/199/commits/6ba2f718c1eb05b86d280068be46d54865dcc04b) and I hope that's what you meant, I explain how it works below:
In this test, I create two different CopyPayload objects with distinct references, namely ref1 and ref2. I then insert the object with reference ref1 twice and the one with reference ref2 once into the database. After the insertion, I verify that the count of copies for each reference matches the expected values: 2 for ref1 and 1 for ref2.

Reply:
Also.. While creating and testing the functionality of the SQLiteJournalFRCopyPayloadRepository, I noticed some incorrect behavior when trying to add two identical ftJournalFRCopyPayload objects with the same reference. I expected this operation to throw an exception, but instead I received a data parsing error.

To understand this behavior and ensure that the repository behaves correctly in such situations, [I added a dedicated unit test](https://github.com/fiskaltrust/middleware/pull/199/commits/79c04726ee0acdfe20254f730c05038d06138390). This test tries to insert two identical objects into the database and checks whether the expected System.Data.DataException exception is thrown.

This test ensures that the repository correctly handles situations where we try to add duplicates, and that an appropriate exception is thrown for invalid operations.

Reply:
Also this test file should be refactored, what I already doing :D